Automattic/wp-calypso
# JP Remote Connect: Make invalid credentials notice clearer

Fixes #24279.

**Before**

<img width="250" alt="screen shot 2018-04-23 at 16 20 56" src="https://user-images.githubusercontent.com/7767559/39136095-644ab53a-4712-11e8-842e-844d4a7b9e4f.png"><img width="250" alt="screen shot 2018-04-23 at 16 20 37" src="https://user-images.githubusercontent.com/7767559/39136109-67fafe24-4712-11e8-8ddf-6a94d4f1e42f.png">

**After**

<img width="250" alt="screen shot 2018-04-23 at 16 19 39" src="https://user-images.githubusercontent.com/7767559/39136143-7929566e-4712-11e8-881e-a60e8bc29c62.png"><img width="250" alt="screen shot 2018-04-23 at 16 20 02" src="https://user-images.githubusercontent.com/7767559/39136150-7b301d80-4712-11e8-9da0-1c7b3d5192e0.png">

## Testing
* Go to https://calypso.live/jetpack/connect?branch=update/jp-remote-install/creds-notice
* Enter the URL of a site without jetpack installed
* On the credentials screen, enter invalid password/user
